enterprise-suite/console-operator/deploy/crds/console.lightbend.com_consoles_crd.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
I'm confused.  How does this file relate to the renamed 
`enterprise-suite/console-operator/deploy/crds/console_v1alpha1_console_crd.yaml → operator/manifests/crds/console_v1alpha1_console_crd.yaml`
below?  They look to be essentially the same thing.  (That plural `_consoles` on the end of this filename threw me btw.)

(Also, I note that `operator/manifests/crds/console_v1alpha1_console_crd.yaml` specifies both a `version` and `versions`.  I think you got rid of the `version` line (from somewhere) in stuff I reviewed earlier.  Not sure if you meant it to be gone now or not.)